src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -633,6 +633,16 @@ def check_organization_by_slug(self, *, slug: str, only_visible: bool) -> Option
 
         return None
 
+    def check_organization_by_id(self, *, id: int, only_visible: bool) -> bool:
+        # See RegionOrganizationCheckService below
+        org_mapping = OrganizationMapping.objects.filter(organization_id=id).first()
```

Comment:
It'd be nice to have this implementation use `OrganizationMapping.objects.get(organization_id=id)` and `except OrganizationMapping.DoesNotExist` to match the other implementation - not necessary though since it works fine as-is